python/numeric-python.ipynb

```diff
@@ -61,9 +61,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "- Grunddatentyp von Numpy: das Array\n",
-    "- Kann man sich als effizientere Liste vorstellen\n",
-    "- Idee von Numpy: Man kann ein Array ähnlich wie eine Zahl verwenden. Operationen werden dann auf allen Elementen ausgeführt\n",
+    "- Grunddatentyp von Numpy ist das **Array**\n",
```

Comment:
Präzise sein? `numpy.ndarray` = n-dimensional array?

Reply:
Von mir aus gern. In der vorherigen Version war das alles recht lopidar formuliert. Ich ging davon aus, dass das bewusst so formuliert wurde um die Leute nicht zu überfordern. Aber meinetwegen kann ich es auch präziser formulieren.